packages/bsky/src/data-plane/server/subscription/index.ts

```diff
@@ -270,6 +276,8 @@ function getMessageDetails(msg: Message):
     return { seq: msg.seq, repo: msg.did, message: msg }
   } else if (message.isIdentity(msg)) {
     return { seq: msg.seq, repo: msg.did, message: msg }
+  } else if (message.isAccount(msg)) {
+    return { seq: msg.seq, repo: msg.did, message: msg }
```

Comment:
I think there's one more of these next door in `subscription/util.ts` within the implementation for `loggableMessage()`!

Reply:
ah good snag 👍